modular_robot_simulation/revolve2/modular_robot_simulation/_test_robot.py

```diff
@@ -0,0 +1,34 @@
+from revolve2.ci_group.simulation import make_standard_batch_parameters
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.body.base import Body
+from revolve2.modular_robot.brain.dummy import BrainDummy
+from revolve2.modular_robot_simulation import ModularRobotScene
+from revolve2.simulation.simulator import Simulator
+
+from ._simulate_scenes import simulate_scenes
+from ._terrain import Terrain
+
+
+def test_robot(
+    robot: ModularRobot | Body, terrain: Terrain, simulator: Simulator
+) -> None:
+    """
+    Test a robot with a manual brain.
+
+    :param robot: The ModularRobot or Body instance.
+    :param terrain: The terrain to test on.
+    :param simulator: The simulator.
+    """
+    if isinstance(robot, Body):
+        body = robot
+        brain = BrainDummy()
+        robot = ModularRobot(body=body, brain=brain)
+
+    batch_parameters = make_standard_batch_parameters()
```

Comment:
this library does not depend in ci-group, so you will have to just write out some parameters here instead